prep-javascript-functions/main.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
Please delete these and address the linter's complaints. the linter is trying to prevent you from accidentally polluting the global scope by omitting the `var` keyword or creating extra variables that aren't being used.